mavros_extras/src/plugins/mocap_fake_gps.cpp

```diff
@@ -61,25 +61,26 @@ class MocapFakeGPSPlugin : public plugin::PluginBase
 	double gps_eph;
 	double gps_epv;
 	ros::Time last_pos_time;
-	double gps_period;	
+	double gps_period;
 
 	/* -*- low-level send -*- */
-	void gps_pose_send
-		(uint64_t time_usec, int32_t lat, int32_t lon, int32_t alt, uint16_t eph, uint16_t epv,
-		 uint16_t vel, int16_t vn, int16_t ve, int16_t vd, uint16_t cog, uint8_t fix_type, uint8_t satellites_visible)
+	void gps_pose_send(uint64_t time_usec, int32_t lat, int32_t lon, int32_t alt,
+				uint16_t vel, int16_t vn, int16_t ve, int16_t vd, uint16_t cog,
+				uint16_t eph = 2, uint16_t epv = 2,
+				uint8_t fix_type = 3, uint8_t satellites_visible = 5)
```

Comment:
Not sure why to add optional args? I more like previous argument list close to message definition.